pwa-shell/src/App.js

```diff
@@ -68,50 +67,49 @@ const routesData = [
         classNames: 'content-list',
         Component: ContentListContainer,
       },
-    ]     
-  },  
+    ],
+  },
 ];
 
 const routes = routesData.map(route => (
   <Route exact key={route.path} path={route.path}>
-    {
-      props => (
-        <>
-          { 
-            route.cssTransitions.map(({timeout, classNames, Component}, index) => (
-              <CSSTransition
-                key={`${route.path}_${index}`}
-                in={props.match && props.match.isExact}
-                unmountOnExit
-                timeout={timeout}
-                classNames={classNames}
-              >
-                <div className="App__page-wrapper">
-                  <Component {...props} />
-                </div>
-              </CSSTransition>                
-            ))
-          }
-        </>
-      )
-    }        
+    {props => (
+      <>
+        {route.cssTransitions.map(
+          ({ timeout, classNames, Component }, index) => (
+            <CSSTransition
+              key={`${route.path}_${index}`}
+              in={props.match && props.match.isExact}
+              unmountOnExit
+              timeout={timeout}
+              classNames={classNames}
+            >
+              <div className="App__page-wrapper">
+                <Component {...props} />
+              </div>
+            </CSSTransition>
+          ),
+        )}
+      </>
+    )}
   </Route>
 ));
 
-const App = () => (  
+const App = () => (
   <IntlProvider
     locale={appLocale}
     defaultLocale="en"
     key={appLocale}
     messages={locales[appLocale]}
-  >    
+  >
     <StateProvider store={store}>
       <HomePageHead />
-      <NetworkStatusContainer>        
-        <ApiManager store={store} lang={appLocale}>
+      <NetworkStatusContainer>
+        <AppLanguage store={store} lang={appLocale} />
```

Comment:
`IntlProvider` and `AppLanguage` share same responsibilities.
I'd suggest, following the approach in `app-builder` repo, to use `StateProvider` as first child of `App`, then have an `IntlProviderContainer` that covers `AppLanguage`'s responsibilities and acts as container for `IntlProvider`. 

Doing this way you will have no more need to pass down the store and you can even remove store from `ApiManager`'s props.

Reply:
good idea @gidesan . I'll change asap

Reply:
Beware:
- `store` is still passed to `ApiManager` and `IntlProviderContainer`
- `IntlProviderContainer` is not acting as a container component: have a look at other containers in the codebase

Reply:
I'm not sure how we can access the `store` object itself in a `container`. Because inside the `ApiManager` needs the `store` object itself to configure with our `@entando/apimanager`. I would suggest to remove the `<ApiManager>` component and just import an init file which configures PWA with `@entando/apimanager` just like what we did in app-builder. What do you think?

Reply:
Also, updates has been commited. `IntlProviderContainer` has been renamed to `AppIntlProviderContainer` so I can create a component `AppIntlProvider` (to avoid conflict with `IntlProvider` in `react-intl`) which basically triggers `setLanguage` method when mounted.

Reply:
You're right about the `ApiManager` usage of `store`. Regarding other stuff, I pushed some changes and we can consider this task ended. Thanks a lot :)